ServiceWebsite/ServiceWebsite.AcceptanceTests/Hooks/DataSetUpTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.Extensions.Configuration;
+using NUnit.Framework;
+using ServiceWebsite.AcceptanceTests.Helpers;
+
+namespace ServiceWebsite.AcceptanceTests.Hooks
+{
+    public class DataSetUpTests
+    {
+        private DataSetUp _dataSetUp;
```

Comment:
Not quite sure why we have tests for our tests. If we're loading clients/secrets in our hooks, it would be safer to fail fast and assert the value(s) are present before attempting to run the test. Say all of our tests require config to be present, assert on the setup so all tests fail at the same point rather than have a test to test the setup fail. I hope that makes sense.

Reply:
these tests are meant to run before the acceptance tests are kicked off. Acceptance tests take longer to start so having integration tests would allow us to have a quicker turnaround. Besides we might also want to test more complex logic in our framework, so they're not necessarily tests for our tests, but for out framework.